pkgs/development/python-modules/codepy/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytools
+, numpy
+, appdirs
+, six
+, cgen
+}:
+
+buildPythonPackage rec {
+  pname = "codepy";
+  version = "2019.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "384f22c37fe987c0ca71951690c3c2fd14dacdeddbeb0fde4fd01cd84859c94e";
+  };
+
+  buildInputs = [ pytools numpy appdirs six cgen ];
```

Comment:
Those are also wrong. numpy is at least propagated and please check the others.

Reply:
Aye sir!

Reply:
I was only able to remove numpy, and I also had to turn appdirs into propagatedBuildInput.